centaur/src/main/resources/integrationTestCases/green/arrays/arrays.inputs

```diff
@@ -1,39 +1,33 @@
-"MultiSampleArrays.callset_name": "callset_name",
-"MultiSampleArrays.samples_fofn": "gs://broad-gotc-test-storage/arrays/multisample/psych-chip-validation/vcf/101342370027/multisample_vcf_fofn",
-"MultiSampleArrays.sample_indices_fofn": "gs://broad-gotc-test-storage/arrays/multisample/psych-chip-validation/vcf/101342370027/multisample_vcf_index_fofn",
-"MultiSampleArrays.disk_size": 100,
-"MultiSampleArrays.preemptible_tries": 0,
-"END": "This line does not need a comma."
-}
+  "Arrays.sample_alias": "NA12716",
+  "Arrays.analysis_version_number": 1,
+  "Arrays.call_rate_threshold": 0.98,
+  "Arrays.reported_gender": "Not Reported",
+  "Arrays.idat_dir_name": "101342370134_R01C02",
+
+  "Arrays.file_of_idat_filenames": "gs://broad-gotc-test-storage/arrays/single_sample/idats/101342370134_R01C02.idats.txt",
+  "Arrays.fingerprint_genotypes_vcf_file": "gs://broad-gotc-test-storage/arrays/single_sample/inputs/101342370134_R01C02.reference.fingerprint.vcf.gz",
+  "Arrays.fingerprint_genotypes_vcf_index_file": "gs://broad-gotc-test-storage/arrays/single_sample/inputs/101342370134_R01C02.reference.fingerprint.vcf.gz.tbi",
+  "Arrays.params_file": "gs://broad-gotc-test-storage/arrays/single_sample/inputs/params.txt",
+
+  "Arrays.bead_pool_manifest_file": "gs://broad-gotc-test-storage/arrays/metadata/PsychChip_v1-1_15073391_A1/PsychChip_v1-1_15073391_A1.bpm",
+  "Arrays.extended_chip_manifest_file": "gs://broad-gotc-test-storage/arrays/metadata/PsychChip_v1-1_15073391_A1/PsychChip_v1-1_15073391_A1.1.2.extended.csv",
+  "Arrays.cluster_file": "gs://broad-gotc-test-storage/arrays/metadata/PsychChip_v1-1_15073391_A1/PsychChip_v1-1_15073391_A1_ClusterFile.egt",
+  "Arrays.zcall_thresholds_file": "gs://broad-gotc-test-storage/arrays/metadata/PsychChip_v1-1_15073391_A1/thresholds.7.txt",
+
+  "Arrays.control_sample_vcf_file" : "gs://broad-gotc-test-storage/arrays/controldata/NA12716.vcf.gz",
+  "Arrays.control_sample_vcf_index_file" : "gs://broad-gotc-test-storage/arrays/controldata/NA12716.vcf.gz.tbi",
+  "Arrays.control_sample_intervals_file" : "gs://broad-gotc-test-storage/arrays/controldata/NA12716.interval_list",
+  "Arrays.control_sample_name" : "NA12716",
+  "Arrays.disk_size": 100,
+
+  "Arrays.ref_fasta": "gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta",
+  "Arrays.ref_fasta_index": "gs://broad-references/hg19/v0/Homo_sapiens_assembly19.fasta.fai",
+  "Arrays.ref_dict": "gs://broad-references/hg19/v0/Homo_sapiens_assembly19.dict",
+  "Arrays.dbSNP_vcf": "gs://broad-references/hg19/v0/dbsnp_138.b37.vcf.gz",
+  "Arrays.dbSNP_vcf_index": "gs://broad-references/hg19/v0/dbsnp_138.b37.vcf.gz.tbi",
+  "Arrays.haplotype_database_file": "gs://broad-references-private/hg19/v0/Homo_sapiens_assembly19.haplotype_database.txt",
+  "Arrays.variant_rsids_file": "gs://broad-references-private/fingerprinting_variant_rsids_v1.list",
+  "Arrays.preemptible_tries": 3
+}
```

Comment:
newline marker here. Khalid pointed this out to me recently - you might want to do the opposite to what this post suggests: https://intellij-support.jetbrains.com/hc/en-us/community/posts/207071445-On-save-new-line-added-to-end-of-file-How-stop-this-